steward/src/utils.rs

```diff
@@ -88,3 +88,10 @@ pub async fn get_eth_provider() -> Result<Provider<Http>, Error> {
 pub fn sp_call_error(message: String) -> Error {
     ErrorKind::SPCallError.context(message).into()
 }
+
+pub fn sp_call_parse_address(address: String) -> Result<H160, Error> {
+    match address.parse::<H160>() {
```

Comment:
I've seen this a bunch, why do we use the `H160` type for address rather than the ethers `Address` type with more support functions? Does the parse for `Address` contain further validation that `H160` might not entail?

Reply:
Just habit. H160 was used early on in `cellar_rebalancer` development and I'm stuck in my ways. Will make the switch.